src/x/ident/identifier_pool_test.go

```diff
@@ -93,10 +93,10 @@ func (s idPoolTestSuite) TestPoolBinaryID() {
 
 	bid := s.pool.BinaryID(v)
 	s.Require().Equal(1+nr, v.NumRef())
+	v.DecRef()
```

Comment:
@robskillington Both of these tests had a `Finalize` while num refs > 0 issue and also read after free issue. 

I removed the check that attempts to read `.Bytes()` not sure why it was testing that in the first place?

Reply:
Sounds good. Yeah that does seem weird.